server/common/src/main/java/org/eclipse/lsp/cobol/common/mapping/MappingService.java

```diff
@@ -205,7 +205,7 @@ private List<MappingItem> buildLocalityMap(TextTransformations textTransformatio
       result.addAll(map);
     }
 
-    String[] originalLines = textTransformations.getText().split("\\r?\\n", -1);
+    String[] originalLines = textTransformations.getText().split("\r?\n", -1);
```

Comment:
Have a doubt, 

- should we use system.lineseparator()? or 

- it doesn't matter because we provide the line separator system property as \r\n while starting the server ([reference link](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/ec3aef28f71fb148b189f8897be915f2e270850c/clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts#L139)
in this case, it should always be \r\n

Reply:
It's for source code, it can be any line separator, so probably `\r?\n` is the best what can we have here.

Reply:
I will return it back :-)